tests/billing_groups_test.js

```diff
@@ -11,82 +11,72 @@ const resource_endpoint = "/billing_groups",
 
 const prism = new Prism(specFile, lobUri, process.env.LOB_API_TEST_TOKEN);
 const payload = {
-  name: "Open Api test",
-  description: "Open Api test",
+  name: "OpenAPI Test",
+  description: "OpenAPI testing for Billing Groups",
 };
-let create = null;
 
 // contract tests happy path
-test("create, list, read, replace, update, then delete a billing_group", async function (t) {
-  create = await prism
-    .setup()
+test.serial.before("create a billing_group", async function (t) {
+  t.plan(3);
+  // JSON.stringify(slackError, null, 2)
+  const create = await prism
+    .setup({ errors: false })
     .then((client) =>
       client.post(resource_endpoint, payload, { headers: prism.authHeader })
     );
   t.assert(create.status === 200);
   t.assert(create.data.name === payload.name);
   t.assert(create.data.description === payload.description);
 
+  t.context.create_id = create.data.id;
+});
+
+test.serial.before("list billing_groups", async function (t) {
+  t.plan(1);
   const list = await prism
-    .setup()
+    .setup({ errors: false })
     .then((client) =>
       client.get(resource_endpoint, { headers: prism.authHeader })
     );
   t.assert(list.status === 200);
+});
 
-  const read = await prism.setup().then((client) =>
-    client.get(`${resource_endpoint}/${create.data.id}`, {
+test.serial.before("read a billing_group", async function (t) {
+  t.plan(3);
+  const read = await prism.setup({ errors: false }).then((client) =>
+    client.get(`${resource_endpoint}/${t.context.create_id}`, {
       headers: prism.authHeader,
     })
   );
   t.assert(read.status === 200);
-  t.assert(create.data.name === payload.name);
-  t.assert(create.data.description === payload.description);
+  t.assert(read.data.name === payload.name);
+  t.assert(read.data.description === payload.description);
+});
 
-  const replace = await prism.setup().then((client) =>
+test.serial.before("update a billing_group", async function (t) {
+  t.plan(2);
+  const update = await prism.setup({ errors: false }).then((client) =>
     client.post(
-      `${resource_endpoint}/${create.data.id}`,
+      `${resource_endpoint}/${t.context.create_id}`,
       {
         name: "Update Open Api Test",
       },
       { headers: prism.authHeader }
     )
   );
-  t.assert(replace.status === 200);
-  t.assert(replace.data.name === "Update Open Api Test");
+  t.assert(update.status === 200);
+  t.assert(update.data.name === "Update Open Api Test");
 });
 
```

Comment:
We might need to delete this part. Idk if creating a billing group with a test token will actually cause sth permanent to spawn associated to our acc. And since there's no publicly available delete endpoint for billing grps, it might be best to leave this test out of this repo (maybe put it in lob-adminapi?).

Reply:
You know, I wondered about that too. I'm going to poke around a little and see if we can automate the deletion of billing groups from our account somehow, but if we can't then I support moving this test to Lob AdminAPI. It would be super cool if we could reference the billing groups endpoint from OpenApi _within_ AdminAPI; I think I've seen something to that effect in the OpenAPI docs before, so we might be able to do it!